amg.c

```diff
@@ -598,19 +670,32 @@ main( hypre_int argc,
 
       FOM2 = cum_nnz_AP / wall_time;
 
+#ifdef USE_CALIPER
+      CALI_MARK_BEGIN("setup-FOM");
+#endif
       if (myid == 0)
       {
          hypre_printf ("\nFOM_Setup: nnz_AP / Setup Phase Time: %e\n\n", FOM2);
       }
 
+#ifdef USE_CALIPER
+      CALI_MARK_END("setup-FOM");
+      adiak_namevalue("Setup-FOM", adiak_general, NULL, "%f", FOM2);
```

Comment:
Then here FOM2 will also need to be replaced by FOM1.

Reply:
Done